RELEASE_PROCESS.md

```diff
@@ -0,0 +1,57 @@
+# Release Process
+
+## General Information
+
+The content of a branch can be released by GitHub `release` action. The name of the release is based on the content of
+the file [`VERSION`](./VERSION). During development, the content of this file is the complete release name of the
+release currently under development and the suffix `-dev` (e.g. `0.1.0-dev`). The content of this file is used for
+generating the version information compiled into the ocm executables.
+
+If a release is created, the `-dev`-suffix is removed and an optional pre-release suffix is appended to generate the
+name of the release (e.g. `0.1.0` or `0.1.0-alpha1`) and to prepare a commit for the release which is used to create a
+release tag.
+
+Additionally, this commit will also add a new release note file under [./docs/releasenotes](./docs/releasenotes). It is
+generated from the appropriate draft release with the basic release name (e.g. `0.1.0`).
+After the release is done, for final releases, a new commit is created to prepare the development of the next release
+by adapting the [`VERSION`](./VERSION) file again. Thereby, if the patch level is 0 (e.g. `0.1.0`), the minor
+version number is increased (e.g. `0.2.0`). If the patch level is not zero (e.g. `0.1.1`), the patch level is increased
+(e.g. `0.1.2`). This commit is pushed to the branch for which you created the release, and it therefore contains the
+release commit in its history.
```

Comment:
I find these sentences a bit confusing.

Is our release version number automatically taken from that file? If so, when is the minor version number increased, if the patch level was once greater zero? How is the patch-level increased?

Reply:
Okay, I see that it is explained some sentences below. However, at first sight this sounds confusing imo. I would probably omit these sentences here and append them below, when the respective release-process is explained